Calculator/index.js

```diff
@@ -1,8 +1,105 @@
 function addEventListenerToButton(element){
     element.addEventListener('click', function(){
-        console.log(element.getAttribute("data-button-value"))
+        let value=element.getAttribute("data-button-value")
+        let type=element.getAttribute("data-button-type")
+
+        switch(type){
+            case "clear": 
+
+                clear();
+                document.getElementById("old-inputs").innerHTML = '';
+                document.getElementById("inputs").innerHTML = '';
+            break;
+
+            case "delete":
+                del();
+            break;
+            
+
+            case "operator":
+
+                numbers[1]=value;
```

Comment:
In the same sense as above, I think you could encapsulate all the logic regarding this function (... and the other ones, of course) in its own separate function